src/erc20ConversionProxy.ts

```diff
@@ -1,6 +1,6 @@
 import { BigDecimal, log } from "@graphprotocol/graph-ts";
 import { TransferWithConversionAndReference } from "../generated/ERC20ConversionProxy/ERC20ConversionProxy";
-import { TransferWithReferenceAndFee } from "../generated/ERC20FeeProxy/ERC20FeeProxy";
+import { TransferWithReferenceAndFee } from "../generated/ERC20FeeProxy_0_2_0/ERC20FeeProxy_0_2_0";
```

Comment:
The content of this does not change. Only the name changes (the code generation keeps the oldest version).